app-connector-client/server/services.js

```diff
@@ -8,38 +8,55 @@ const connection = require("./connection")
 const OAUTH = "/authorizationserver/oauth/token"
 const METADATA = "/metadata"
 const request = require("request-promise")
+var success_count = 0;
+var failed_count = 0;
+var apis_success = [];
+var apis_failed = [];
 module.exports = {
     createServicesFromConfig: createServicesFromConfig,
     getAllAPI: getAllAPI,
     getAllAPIs, getAllAPIs,
     createAPI: createAPI,
     updateAPI: updateAPI,
-    fillServiceMetadata: fillServiceMetadata
+    fillServiceMetadata: fillServiceMetadata,
+    getStatus: getStatus,
+    createEventsFromConfig: createEventsFromConfig,
+    fillEventData: fillEventData
 }
 
 async function createServicesFromConfig(hostname, apisConfig, registeredApis) {
     if (!apisConfig)
         return
 
     var error_message = ""
+    success_count = 0;

```

Comment:
why the variables are getting rest? I thought we manage the global state only for a bulk registration., but this method here is doing a single registration only.

Reply:
no this is bulk registration method??